model/workflow.go

```diff
@@ -508,25 +508,17 @@ type End struct {
 
 // UnmarshalJSON ...
 func (e *End) UnmarshalJSON(data []byte) error {
-
-	endMap := make(map[string]json.RawMessage)
-	if err := json.Unmarshal(data, &endMap); err != nil {
-		e.Terminate = false
-		e.Compensate = false
-		return nil
-	}
-
-	if err := unmarshalKey("compensate", endMap, &e.Compensate); err != nil {
```

Comment:
why removing these?

Reply:
`primitiveOrStruct` now takes care of it. If it's not a map, `end` is false, otherwise, every other key is unmarshaled.